src/main/java/com/netgrif/application/engine/files/local/LocalStorageService.java

```diff
@@ -0,0 +1,86 @@
+package com.netgrif.application.engine.files.local;
+
+import com.netgrif.application.engine.files.interfaces.IStorageService;
+import com.netgrif.application.engine.files.throwable.BadRequestException;
+import com.netgrif.application.engine.files.throwable.ServiceErrorException;
+import com.netgrif.application.engine.files.throwable.StorageException;
+import com.netgrif.application.engine.petrinet.domain.dataset.StorageField;
+import com.netgrif.application.engine.workflow.domain.FileStorageConfiguration;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.*;
+
+@Slf4j
+@Service
+public class LocalStorageService implements IStorageService {
+    FileStorageConfiguration fileStorageConfiguration;
+
+    @Autowired
+    public void setFileStorageConfiguration(FileStorageConfiguration fileStorageConfiguration) {
+        this.fileStorageConfiguration = fileStorageConfiguration;
+    }
+
+    @Override
+    public String getType() {
+        return "LOCAL";
```

Comment:
StorageType enum can be used instead of string constant.

Reply:
Done